activerecord/lib/active_record/migration.rb

```diff
@@ -277,15 +277,7 @@ def initialize
   #
   # == About the schema_migrations table
   #
-  # Rails versions 2.0 and prior used to create a table called
-  # <tt>schema_info</tt> when using migrations. This table contained the
-  # version of the schema as of the last applied migration.
-  #
-  # Starting with Rails 2.1, the <tt>schema_info</tt> table is
-  # (automatically) replaced by the <tt>schema_migrations</tt> table, which
-  # contains the version numbers of all the migrations applied.
-  #
-  # As a result, it is now possible to add migration files that are numbered
+  # It is possible to add migration files that are numbered
```

Comment:
This sounds weird as an opening sentence under the sub-heading.

However, I think this whole paragraph is really still describing how the 2.1+ solution fixes a shortcoming in the old approach... so it can probably go too.